core/templates/pages/contributor-dashboard-page/modal-templates/translation-modal.directive.html

```diff
@@ -36,7 +36,7 @@
     <div class="oppia-translation-editor-section">
       <div class="protractor-test-state-translation-editor">
         <ck-editor-copy-toolbar></ck-editor-copy-toolbar>
-        <schema-based-editor schema="HTML_SCHEMA" local-value="activeWrittenTranslation.html">
+        <schema-based-editor schema="HTML_SCHEMA" local-value="activeWrittenTranslation.html" dir="<[targetLocaleDirection]>">
```

Comment:
Can you use "direction" for the attribute name, too?

Also, where is this "dir" used? I don't see a component that takes it as the input in this PR, so I am a bit surprised that this has any effect. Did this work when you tested it manually? If you remove the "dir" attribute does it still work?

Reply:
This is built in, `dir` is an html attribute specifically for text direction

Reply:
See: https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/dir
For more details

Reply:
Ah, I see -- interesting that it works on custom directives too. OK, thanks!